tests/ui/manual_split_once.stderr

```diff
@@ -0,0 +1,34 @@
+error: manual implementation of `split_once`
+  --> $DIR/manual_split_once.rs:13:13
+   |
+LL |     let _ = "key=value".splitn(2, '=').nth(0).unwrap();
+   |             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ help: try this: `"key=value".split_once('=').unwrap().0`
```

Comment:
This suggestion isn't the same in general.
`"no_equals".splitn(2, '=').nth(0).unwrap()` returns `"no_equals"`.
`"no_equals".split_once('=').unwrap().0` panics.

Reply:
`map_or` is suggested now.